packages/webapp/pages/squads/[handle]/[token].tsx

```diff
@@ -83,9 +83,9 @@ const SquadReferral = ({ token, handle }: SquadReferralProps): ReactElement => {
   );
 
   const onJoinClick = async () => {
-    if (!loggedUser) return showLogin('join squad');
+    if (loggedUser) return onJoinSquad();
 
-    return onJoinSquad();
+    return showLogin('join squad', { referral: member.user.id });
```

Comment:
Don't think the order is right, this was just a test I think?

Reply:
This should be right. I switched as I think there could be additional things we do under this object in the future. We can revert though.

Reply:
Hmm but now once you login, it has no callback right?
Where before it logged in, then do whatever you invoked.

Or am I understanding it wrong?

Reply:
Oh, the current setup would make the user have to click the "Join Squad" again. Let me make it automatic.